releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v5.1.0 - January, 07 2022
+* Added support for sovereign clouds such as Azure Government and Azure China.
+* Updated task input from **App Configuration Name** to **App Configuration Endpoint**.
```

Comment:
This is a breaking change, right? We should mark it as breaking change as we did before. Any instructions we could provide for users to upgrade from v4.* to v5.*?

Reply:
Yes, we should mark it as a breaking change, let me update that. For the instructions, how about?
**Before**: Task input was **App Configuration Name**
**After**: Task input updated to App Configuration endpoint, the  **App Configuration Endpoint** can be gotten from the App configuration store overview page.